parenthetic/test_parenthetics.py

```diff
@@ -0,0 +1,74 @@
+from __future__ import unicode_literals
+import pytest
+from parenthetics import parenthetical
+
+
+#  Case that should return -1
+broken_case = [
+    "))sdf)as((43215("
+    ")tqw()3",
+    "345a)))",
+    "eq()()q()hq()(hqre[][][]{()))",
+    ")dfh",
+    "ehu{()()())()()()()",
+    "({})()()()()))asdg(())asgwq)()321t5(((()12fds"]
+
+#  Case that should return 1
+open_case = [
+    "!@#^$#&(324643)(",
+    "()asdgw()(@!#&^$#&)(",
+    "13246((()asd()oigo",
+    "$@*-=()()(",
+    "(qy1235)()(qwet)((",
+    "()()((3461(()25()153()145"
+]
+
+#  Case that should return 0
+okay_case = [
+    "(1266)()32164()!$#^%@&()|||",
+    "((qwet)qwet)",
+    "(52135(()1231())()q143265123()())"
+]
+
+#  Types that should fail with TypeError
+bad_types = [
+    0,
+    None,
+    5.32324,
+    {1: 123123, "a": 45243},
+    [1, 2, 3, 4, 5]
+]
+
+
+def test_broken_case_via_assert(broken_case=broken_case):
```

Comment:
This syntax is not really all that helpful to you.  The broken_case symbol is bound in the scope of this function, but only because it's bound in the module as well.  This does not operate the way you are thinking it does.  

I believe what you were looking for is the "fixture" approach we used in the learning journal tests.  Do you remember how that worked?  Can you apply the same approach here?